src/AlgebraicGeometry/Surfaces/EnriquesAut.jl

```diff
@@ -0,0 +1,908 @@
+################################################################################
+# Computation of Automorphism groups of Enriques surfaces following
+# https://doi.org/10.1307/mmj/20195769
+# and
+# http://www.arxiv.org/abs/1909.10813
+# Initial Version by Simon Brandhorst
+################################################################################
+
+@doc raw"""
+    EnriquesBorcherdsCtx
+
+Holds data for computing invariants of families of Enriques surfaces with fixed root 
+invariants, in particular the automorphism group of a general element. 
+
+The setting is the following:
+Let ``Y`` be an Enriques surface over an algebraically closed field of characteristic not ``2``
+and ``π : X → Y`` its universal covering K3 surface. We denote by ``S_Y`` and ``S_X`` the 
+numerical lattices of ``Y`` and ``X``.
+
+Most importantly the type stores a chain of lattices
+
+```math
+S_Y(2) \subseteq S_X \subseteq L_{1,25}
+```
+
+where ``L_{1,15}`` is an even unimodular lattice of signature $(1,25)``. 
```

Comment:
```suggestion
where ``L_{1,15}`` is an even unimodular lattice of signature ``(1,25)``. 
```
formatting is off, maybe fix like this?